mail_show_follower/models/mail_mail.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2020 Valentin Vinagre <valentin.vinagre@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class MailMail(models.Model):
+    _inherit = "mail.mail"
+
+    def _send(self, auto_commit=False, raise_exception=False, smtp_session=None):
```

Comment:
Please check all the original function correctly...
Make the relevant changes.

https://github.com/odoo/odoo/blob/44057dac2127fbb26d770544a25af49d0e737b24/addons/mail/models/mail_mail.py#L581-L749

Reply:
Checked.

Reply:
I made changes. Can you review again pls? THX